src/main/scala/com/databricks/spark/sql/perf/mllib/MLBenchmarks.scala

```diff
@@ -0,0 +1,39 @@
+package com.databricks.spark.sql.perf.mllib
+
+import com.databricks.spark.sql.perf.mllib.classification.LogisticRegression
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.SQLContext
+
+import com.databricks.spark.sql.perf.{ExtraMLTestParameters, MLTestParameters}
+import OptionImplicits._
+
+case class MLBenchmark(
```

Comment:
I'd prefer to rename this since "Benchmark" has a special meaning already in spark-sql-perf.  Maybe we can call this concept a "Test" instead of a "Benchmark."